### PR TITLE
#PLAT-2236 Non self-service challenge should also get a forum

### PR DESCRIPTION
**Pull request by himaniraghav3**
Removed the conditions for only selfservice challenges to create a forum. For challenges that don't have discussions, we are anyway not creating forums.

### CODE REVIEW

**Review by ThomasKranitsas — CHANGES_REQUESTED** [review]
Please remove any unused code. Do not comment it out

**Review by ThomasKranitsas — CHANGES_REQUESTED** [review]
Please remove the semi-colons `;` to keep the consistency in the code

File: src/services/vanilla.js [hunk 1]
```diff
@@ -295,9 +289,9 @@ async function updateVanillaGroup (challenge) {
 
   const { body: groups } = await vanillaClient.searchGroups(challenge.id)
   if (groups.length === 0) {
-    const isSelfService = challenge.legacy.selfService && challenge.legacy.selfService === true ? true: false
+    //const isSelfService = challenge.legacy.selfService && challenge.legacy.selfService === true ? true: false
```
Comment (ThomasKranitsas) [inline src/services/vanilla.js:292]: Please remove this one as well

**Review by ThomasKranitsas — CHANGES_REQUESTED** [review]

File: src/services/vanilla.js (hunk 1 above)
Comment (himaniraghav3) [reply src/services/vanilla.js]: Done

**Review by ThomasKranitsas — APPROVED** [review]

**Pull request merged**